app/views/session.py

```diff
@@ -35,15 +36,15 @@ def login():
     if session:
         session.clear()
 
-    logger.debug("attempting token authentication")
-    authenticator.jwt_login(request)
-    logger.debug("token authenticated - linking to session")
+    token = decrypt_token(request.args.get('token'))
+    if not use_token(token.get('jti')):
```

Comment:
Due to needing a globally unique identifier that isn't duplicated across multiple servers (negligible probability)